pd_ack_to_nagios_ack_poller.pl

```diff
@@ -78,7 +80,15 @@
    if(!defined($opts{last_id}) &&
       (-e $opts{last_id_file}) && !(-w $opts{last_id_file}));
 die "can't access pipe $opts{nagios_command_pipe}" if(!(-w $opts{nagios_command_pipe}));
-die "--pagerduty_token|-p required" unless($opts{pagerduty_token});
+
+if(!defined($opts{pagerduty_token})) {
+  $opts{pagerduty_token_file} ||= '/etc/nagios/pd_ack_to_nagios_ack_poller.key';
+  open(FILE, $opts{pagerduty_token_file}) || die "can't open pagerduty_token_file, and no token has been passed on the command line";
+  die "pagerduty_token_file is empty, and no token has been passed on the command line"
+    if(-z $opts{pagerduty_token_file});
+  $opts{pagerduty_token} = <FILE>;
+  close(FILE);
```

Comment:
```suggestion
  close(FILE);
  chomp($opts{pagerduty_token});
```
prevents newlines from being added to the token and breaking the subsequent `curl`